onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -2395,6 +2395,7 @@ Status RegisterOnnxOperatorKernels(KernelRegistry& kernel_registry) {
   return Status::OK();
 }
 
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
should this be `#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)`?